Sources/Core/Glob.swift

```diff
@@ -57,7 +57,7 @@ public class Glob: Collection {
 
     public static let defaultBlacklistedDirectories = ["node_modules", "Pods"]
 
-    private var isDirectoryCache = [String: Bool]()
+    private static let isDirectoryCache = NSCache<NSString, NSNumber>()
```

Comment:
Could you please add a comment mentioning that this is faster and why it's an NSCache

Reply:
I chose an NSCache because we're dealing with a cache here. It works with an NSDictionary as well, I can use whatever you think works best. Regarding the reason I can add a comment, although I can only mention that the finding is empirical using Instruments.